src/components/DeckBuilderDryRunView/index.js

```diff
@@ -2,18 +2,52 @@ import React from 'react'
 import modifyDeck from '../../helpers/modifyDeck'
 import DryRunner from '../DryRunner'
 import DeckMechanisms from '../DeckMechanisms'
+import { NotificationContext } from '../NotificationProvider'
 import WikiLink from '../WikiLink'
+import { BRAWLS } from '../../constants/brawl'
 import isCard from '../../helpers/isCard'
+import isSuggestedDeck from '../../helpers/isSuggestedDeck'
 
 export default props => {
   const params = new URLSearchParams(window.location.search)
   // The mode is theoretically not quite supposed to be changed at run time, but
   // this is a workaround to be able to pick an initial hand for testing
   // purposes. The mode is restored to `AUTOMATIC` as soon as the 4th card has
   // been picked.
+  const { notify: sendNotification } = React.useContext(NotificationContext)
+  const notify = React.useCallback(
+    message => sendNotification({ icon: 'sword', children: message }),
+    [sendNotification]
+  )
```

Comment:
You seem to be reversing the naming. Maybe we could do:

```js
   const notify = React.useContext(NotificationContext)
   const sendNotification = React.useCallback(
      message => notify({ icon: 'sword', children: message }),
      [notify]
    )
```

Reply:
Looks like I just copy-pasted this from the CollectionProvider file 😅